pkg/operator/ceph/object/spec_test.go

```diff
@@ -597,3 +597,48 @@ func TestGetDaemonName(t *testing.T) {
 		})
 	}
 }
+
+func TestMakeRGWPodSpec(t *testing.T) {
+	store := simpleStore()
+	info := clienttest.CreateTestClusterInfo(1)
+	data := cephconfig.NewStatelessDaemonDataPathMap(cephconfig.RgwType, "default", "rook-ceph", "/var/lib/rook/")
+	c := &clusterConfig{
+		store:       store,
+		rookVersion: "rook/rook:myversion",
+		clusterSpec: &cephv1.ClusterSpec{
+			CephVersion: cephv1.CephVersionSpec{Image: "quay.io/ceph/ceph:v15"},
+			Network: cephv1.NetworkSpec{
+				HostNetwork: true,
+			},
+		},
+		clusterInfo: info,
+		DataPathMap: data,
+	}
+
+	resourceName := fmt.Sprintf("%s-%s", AppName, c.store.Name)
+	rgwConfig := &rgwConfig{
+		ResourceName: resourceName,
+		DaemonID:     "default",
+	}
+	tests := []struct {
+		name  string
+		isSet bool
+	}{
+		{"hostNetwork is not set to object store", false},
+		{"hostNework is set to object store", true},
+	}
```

Comment:
How about another test where the cluster is not host network, but the object store is host network?

Reply:
if object store is host network, we don't care the host network about the cluster.

Reply:
three testcase is enough:

1. host networking is not set to rgw daemon
2. host networking is set to rgw daemon but not enabled
3. host networking is enabled for the rgw daemon

Reply:
Maybe we only care about the host networking is nil or not nil for the object store.